### DISCUSSION TITLE
Toyota Remote Start through HA

### DISCUSSION THREAD

**Discussion by Dapooch** [DurgNomis-drol/ha_toyota#118] [Q&A]
Hello all!  We just got a 2023 Highlander and I already have it connected through HA.  Love the data that I have been able to seamlessly pull in and being able to lock/unlock through HA app.  

Looking through documentation here, I was not able to see if remote start through HA would be an option.  Is this something that I could set up manually?

Thanks for any help and for all the time you guys have already dedicated to this project!

**Comment by DurgNomis-drol** [comment]
Thanks 😄 

Someone would need to implement it, unfortunately I don't own a car that supports this, so it would need to be someone that does. There is no documentation on Toyota's API, so every thing is done by trial and error

**Reply by JamLaMin** [reply]
@DurgNomis-drol 
Do you already own a new car😛🤪
